packages/next-app/next.config.js

```diff
@@ -12,6 +12,10 @@ const moduleExports = {
   },
   reactStrictMode: true,
   swcMinify: true,
+  webpack: config => {
+    config.resolve.fallback = { fs: false, net: false, tls: false };
+    return config;
+  },
```

Comment:
can you elaborate on why we need this? 🤔

Reply:
this is actually related to rainbowkit migration [3. Ensure bundler and polyfill compatibility](https://www.rainbowkit.com/docs/migration-guide)

Will write a comment about this there